### PR TITLE
Small additions to GETTINGSTARTED.md/Update to Readme.md

### PR DESCRIPTION
**Pull request by Dragonisser**

### CODE REVIEW

File: GETTINGSTARTED.md [hunk 1]
```diff
@@ -44,7 +44,8 @@ DllRelease=Full/Path/To/FalloutTogether.dll
 DllDebug=Full/Path/To/FalloutTogether_d.dll
 ```
 
-For Skyrim and Skyrim SE, you will only need to add the full path in front of the `SkyrimSE.exe` and `.dll`.
+For Skyrim and Skyrim SE, you will only need to add the full path in front of the `SkyrimSE.exe` and `.dll`.
```
Comment (yamashi) [inline GETTINGSTARTED.md:47]: There is only SkyrimSE now

Comment (Dragonisser) [reply GETTINGSTARTED.md:47]: Thought so, wasn't sure if i should remove it or not. Ill get rid of it.

Comment (Dragonisser) [reply GETTINGSTARTED.md]: Changed the text and added additional informations

File: GETTINGSTARTED.md [hunk 2]
```diff
@@ -12,6 +12,12 @@ Run the following command: ``git clone --recursive https://github.com/tiltedphoq
 
 ## Setting up the environment
 
+### Requirements
+
+The following requirements are needed as mentioned in the [Readme](https://github.com/tiltedphoques/TiltedOnline#windows):
+
+You will need [Visual Studio 2019](https://www.visualstudio.com/downloads/) (the community edition is freely available for download) and the [Directx SDK](https://www.microsoft.com/en-us/download/details.aspx?id=6812) to build the project, [Python 3](https://www.python.org/downloads/), [Python 2](https://www.python.org/downloads/) and [Node.js](https://nodejs.org/en/) for the scripts to run properly.
```
Comment (yamashi) [inline GETTINGSTARTED.md:19]: Now that I think of it Directx SDK and python are not required anymore

Comment (Dragonisser) [reply GETTINGSTARTED.md]: Removed it

**Pull request merged**